webpack.config.ts

```diff
@@ -48,16 +47,16 @@ const config: Configuration = {
   module: {
     rules: [
       {
-        exclude: [/node_modules/, /__tests__/, /__mocks__/],
-        test: /\.(jsx?|tsx?)$/,
+        exclude: [/node_modules/u, /__tests__/u, /__mocks__/u],
+        test: /\.(?<temp1>jsx?|tsx?)$/u,
```

Comment:
since it's not accessed maybe converting to non-capturing group makes more sense here
`/\.(?:jsx?|tsx?)$/u`